hack/jenkins/test-flake-chart/report_flakes.sh

```diff
@@ -85,6 +85,10 @@ awk -F, 'NR>1 {
   | sort -g -t, -k2,2 \
   >> "$TMP_FAILED_RATES"
 
+# Filter out arm64 and macOS tests until they're more stable
+grep -v arm64 "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES"
+grep -v macOS ""$TMP_FAILED_RATES" > ""$TMP_FAILED_RATES"
```

Comment:
I'm assuming the extra double quotes are a mistake?

Reply:
yeah i fixed it